crnk-setup/crnk-setup-spring/src/main/java/io/crnk/spring/client/RestTemplateAdapter.java

```diff
@@ -78,7 +82,7 @@ private synchronized void initImpl() {
                 } else if (requestFactory instanceof HttpComponentsClientHttpRequestFactory) {
                     HttpComponentsClientHttpRequestFactory apacheRequestFactory =
                             (HttpComponentsClientHttpRequestFactory) impl.getRequestFactory();
-                    apacheRequestFactory.setReadTimeout(networkTimeout.intValue());
+//                    apacheRequestFactory.setReadTimeout(networkTimeout.intValue());
```

Comment:
Any reason for removing?

Reply:
Ah forgot about this one. The method/property were deprecated:

> As of version 6.0, setting this property has no effect.
To change the socket read timeout, use SocketConfig.Builder.setSoTimeout(Timeout), supply the resulting SocketConfig to org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManagerBuilder.setDefaultSocketConfig(SocketConfig), use the resulting connection manager for org.apache.hc.client5.http.impl.classic.HttpClientBuilder.setConnectionManager(HttpClientConnectionManager), and supply the built HttpClient to HttpComponentsClientHttpRequestFactory(HttpClient).
Deprecated
as of 6.0, in favor of SocketConfig.Builder.setSoTimeout(Timeout), see above.